### PR TITLE
[v9] Move Helm deployment guides

### PR DESCRIPTION
**Pull request by ptgott**
Backports #13105

See #12654

The Kubernetes Access section includes guides for using the Kubernetes
Service as well as guides for running the Auth and Proxy Service on
Kubernetes. This is misleading, since (a) you can run the Auth/Proxy
on Kubernetes without using Kubernetes Access and (b) you can use
Kubernetes Access without running the Auth/Proxy on Kubernetes.

This change focuses on our Helm deployment guides. These guides
are not related to the Kubernetes Service, but rather to deploying
the Auth Service and Proxy Service on Kubernetes. I've suggested
moving these guides to a /setup/helm-deployments section for visibility.

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: docs/config.json [hunk 1]
```diff
@@ -110,6 +110,38 @@
             }
           ]
         },
+        {
+          "title": "Helm Deployments",
+          "slug": "/setup/helm-deployments/",
+          "hideInScopes": "cloud",
```
Comment (klizhentas) [inline docs/config.json:116]: Please remove `hideInScopes`

**Review by klizhentas — CHANGES_REQUESTED** [review]

**Comment by ptgott** [comment]
@klizhentas I've removed the `hideInScopes` fields from `config.json`.

**Comment by ptgott** [comment]
@klizhentas Just following up to see if the changes you requested have now been made

**Comment by ptgott** [comment]
@klizhentas Just following up on this since you requested changes. Thanks!

**Review by klizhentas — APPROVED** [review]

**Pull request merged**